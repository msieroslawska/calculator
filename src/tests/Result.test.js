import React from 'react';
import Result from '../components/Result';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const component = renderer
    .create(<Result value="0" />);
  const tree = component.toJSON();

  expect(tree).toMatchSnapshot();
});