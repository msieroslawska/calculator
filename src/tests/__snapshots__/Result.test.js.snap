// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="result"
>
  0
</div>
`;
